conda/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "dpgen" %}
+{% set version = os.popen('git describe --tags --abbrev=0').read().strip('\n').lstrip('v').replace('-', '_') %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/deepmodeling/dpgen
+  git_rev: v{{ version }}
```

@felix5572 You can consider to use
```suggestion
  path: ..
```
instead. It will fetch source codes from the local file